parsec-evidence-tpm.md

```diff
@@ -1 +1,32 @@
-**TODO:** describe TPM explicit platform attestation
+# TPM Parsec Evidence
+
+This document describes the format of the credential produced by the Parsec
+service as a key attestation credential for TPM-backed keys.
+
+The Parsec service produces a blob of binary data as a response to a key
+attestation operation issued by a client. The service takes in a nonce and uses
+it as freshness for the two attestation tokens it produces.
+
+The outer format of the credential produced by Parsec is a [Conceptual Message
+Wrapper](https://datatracker.ietf.org/doc/draft-ftbs-rats-msg-wrap/), which is
+simply a CBOR-encoded array with the first value identifying the type of the
+second value. In this specific case, the type is represented by the string
+`application/vnd.parallaxsecond.key-attestation.tpm`, a media type string. The
+value is itself a CBOR encoded structure, namely a Combined Attestation Bundle
+(CAB) as defined [here](https://datatracker.ietf.org/doc/draft-bft-rats-kat/).
+The full high-level format is shown below.
+
+```
+parsecTpmAttestation = [
+    "application/vnd.parallaxsecond.key-attestation.tpm",
+    {
+        "kat" => parsecTpmKeyStmtFormat,
+        "pat" => parsecTpmPlatStmtFormat
+    }
+]
```

Comment:
as discussed, this is the split between parsecTpm and CMW payloads:

* Parsec TPM
```
parsecTpmAttestation = {
    "kat" => parsecTpmKeyStmtFormat,
    "pat" => parsecTpmPlatStmtFormat
}
```
* the corresponding CMW
```
[
    "application/vnd.parallaxsecond.key-attestation.tpm"
    bytes .cbor parsecTpmAttestation 
]
```